src/model/mod.rs

```diff
@@ -1 +1,159 @@
-pub mod model;
+pub mod commands;
```

Comment:
Let's not put this in mod.rs. Instead, let's split up into (for now): accounts.rs, assets.rs, peers.rs and put in commands related to accounts, assets, and peers into those three files. This will be a lot cleaner I think.

Reply:
Are you share about that? Rust modules are not the same as packages and namespaces, they are not assume that type of granularity. 
Also, how to be with commands that works with accounts and assets at the same time? `TransferAsset` should provide information about both, account and asset.

Reply:
In my mind, transfer asset, add asset, subtract asset should be with asset. Account registration, signatory commands, permissions should be with account, etc. Putting all commands into one file is going to get somewhat messy very fast, unfortunately.

Reply:
@takemiyamakoto cool, will refactor it so and ask questions about tricky commands